src/StatusQ/Components/StatusExpandableSettingsItem.qml

```diff
@@ -0,0 +1,150 @@
+import QtQuick 2.14
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+Rectangle {
+    id: root
```

Comment:
I know we talk about it but i remember than we agreed use filename id in StatusQ. But maybe im wrong :)

Reply:
old habits die hard @borismelnik :D will change it!